### PR TITLE
Update docs on PlatformColor and themes

### LINKED ISSUE #167: Update AppTheme guidance doc to point to Appearance module instead
The top section of this guidance doc needs to be updated : https://microsoft.github.io/react-native-windows/docs/windowsbrush-and-theme

[AppTheme](https://microsoft.github.io/react-native-windows/docs/apptheme-api) was deprecated in Windows to use [Appearance](https://reactnative.dev/docs/appearance) module instead.

### PR DESCRIPTION
**Pull request by acoates-ms**
Change docs from `windowsbrush` -> `PlatformColor` and `AppTheme` -> `Appearance`

Fixes #167

### CODE REVIEW

File: docs/windowsbrush-and-theme.md [hunk 1]
```diff
@@ -15,48 +15,69 @@ In this example, we'll look at three things:
 - How to switch styles when a theme change has occurred
 - Handling a theme changed event
 
-#### Setting up your app to be sensitive to theme changes
+#### Using hooks to be sensitive to theme changes
 
-First import the Platform API into your React Native app.
+First import the `useColorScheme` hook into your React Native app.
```
Comment (NickGerleman) [inline docs/windowsbrush-and-theme.md:20]: Consider adding a warning (not sure if we have a warning box) that this won't work over web debugging. It's a limitation of core, but it has consistently tripped people up.

Comment (NickGerleman) [reply docs/windowsbrush-and-theme.md:20]: Oh whoops. Wrong section...

File: docs/windowsbrush-and-theme.md [hunk 2]
```diff
@@ -15,48 +15,69 @@ In this example, we'll look at three things:
...
+          <Button title='click me' color={colorScheme === 'dark' ? 'grey' : 'orange'}/>
+        );
+      };
+```
+
+> Note: useColorScheme() will always return 'light' when remote debugging.
+
+#### Setting up your app to be sensitive to theme changes without hooks
+
+First import the `Appearance` API into your React Native app.
```
Comment (NickGerleman) [inline docs/windowsbrush-and-theme.md:39]: Consider adding a warning (not sure if we have a warning box) that this won't work over web debugging. It's a limitation of core, but it has consistently tripped people up.

File: docs/windowsbrush-and-theme.md (hunk 1 above)
Comment (acoates-ms) [reply docs/windowsbrush-and-theme.md:20]: There is a warning box a few lines down.

File: docs/windowsbrush-and-theme.md (hunk 2 above)
Comment (acoates-ms) [reply docs/windowsbrush-and-theme.md:39]: There is a warning box a few lines down.

**Pull request merged**